test/dotnet-new3.UnitTests/dotnet-new3.IntegrationTests.csproj

```diff
@@ -11,6 +11,39 @@
     <PackageReference Include="Wcwidth.Sources" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Common.ProjectTemplates.2.1" Version="5.0" GeneratePathProperty="true" />
```

Comment:
Should we extend it also to other packages frequently used (like web template)?

Reply:
Might make sense yes